build/versions.props

```
@@ -18,6 +18,7 @@
     <HealthCheckAzureStorageBlobs>8.0.1</HealthCheckAzureStorageBlobs>
     <HealthCheckAzureStorageFilesShares>8.0.1</HealthCheckAzureStorageFilesShares>
     <HealthCheckAzureStorageQueues>8.0.1</HealthCheckAzureStorageQueues>
+    <HealthCheckClickHouse>8.0.1</HealthCheckClickHouse>
```

Similarly to other comments, the next version will be 9.0.0
 (https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/2333)

```suggestion
    <HealthCheckClickHouse>9.0.0</HealthCheckClickHouse>
```